lib/librte_hash/rte_hash_version.map

```diff
@@ -53,3 +53,10 @@ DPDK_18.08 {
 	rte_hash_count;
 
 } DPDK_16.07;
+
+EXPERIMENTAL {
+	global:
+
+	rte_hash_iterate;
+	rte_hash_iterate_conflict_entries;
+} DPDK_2.1;
```

Comment:
Notice that `rte_hash_iterate()` is being versioned, not becoming experimental. So `rte_hash_iterate()` must be in `DPDK_18.11` (i.e. a new version) and this version modifies `DPDK_18.08` not `DPDK_2.1`.

`rte_hash_iterate_conflict_entries()` must be alone in an experimental section.

After the change above, we should be able to not change file `lib/librte_hash/Makefile` to add define `ALLOW_EXPERIMENTAL_API`.